packages/record-tuple-polyfill/src/tuple.js

```diff
@@ -116,6 +113,14 @@ Object.defineProperty(Tuple.prototype, "length", {
     },
 });
 
+Tuple.prototype.toString = function toString() {
+    return "[tuple Tuple]";
```

Comment:
The spec instead does Array.prototype.toString(this).

Reply:
The spec text calls a `TupleToString` abstract operation which isn't defined anywhere 🤔

Reply:
https://tc39.es/proposal-record-tuple/ on master it is called `Tuple::toString` and just defers to `%Array.prototype.join%`. https://tc39.es/proposal-record-tuple/#sec-immutable-types-tuple-tostring